infra/build/functions/build_and_run_coverage.py

```diff
@@ -188,6 +188,21 @@ def get_build_steps(  # pylint: disable=too-many-locals, too-many-arguments
         ],
     })
 
+  # Upload the fuzzer logs. Delete the old ones just in case
+  upload_fuzzer_logs_url = bucket.get_upload_url('logs')
+  build_steps.append(build_lib.gsutil_rm_rf_step(upload_fuzzer_logs_url))
```

Comment:
this shouldn't error out if they don't already exist right?

Reply:
Yeah but it wouldn't have in the case I was trying to debug.